dolphinscheduler-ui/src/js/module/i18n/locale/en_US.js

```diff
@@ -449,7 +455,7 @@ export default {
   'Task queue': 'Task queue',
   'Error command count': 'Error command count',
   'Normal command count': 'Normal command count',
-  'Manage': 'Manage',
+  'Manage': ' Manage',
```

Comment:
May I ask a question, why is there an extra space here？

Reply:
> May I ask a question, why is there an extra space here？

![image](https://user-images.githubusercontent.com/28628088/69692358-bc9e6b80-110c-11ea-9211-303d2d34c14e.png)
This field needs to be separated by spaces

Reply:
OK,thanks,understood ,good job.